coalib/results/Diff.py

```diff
@@ -231,7 +231,7 @@ def split_diff(self, distance=1):
 
         >>> diff = Diff.from_string_arrays([     'b', 'c', 'e'],
         ...                                ['a', 'b', 'd', 'f'])
-        >>> len(list(diff.split_diff()))
+        >>> len(list(diff.split_diff(distance=1)))
```

Comment:
Shouldn't you show the output of the default method run (with no arguments) in doctests ?